packer/builder/azure/arm/step_create_resource_group.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
+// All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.
+// See License.txt in the project root for license information.
+
+package arm
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/arm/resources"
+	"github.com/Azure/packer-azure/packer/builder/azure/common/constants"
+	"github.com/mitchellh/multistep"
+	"github.com/mitchellh/packer/packer"
+)
+
+type StepCreateResourceGroup struct {
+	client *AzureClient
+	create func(resourceGroupName string, location string) error
+	say    func(message string)
+	error  func(e error)
+}
+
+func NewStepCreateResourceGroup(client *AzureClient, ui packer.Ui) *StepCreateResourceGroup {
+	var step = &StepCreateResourceGroup{
+		client: client,
+		say:    func(message string) { ui.Say(message) },
+		error:  func(e error) { ui.Error(e.Error()) },
+	}
+
+	step.create = step.createResourceGroup
```

Comment:
Yes, you're right. I was too fast with my judgement...
BTW, a :egg: will no longer yield a :chicken: (baked like that)